exporting/exporting_engine.c

```diff
@@ -11,7 +11,7 @@ void analytics_exporting_connectors_ssl(BUFFER *b)
         for (struct instance *instance = engine->instance_root; instance; instance = instance->next) {
             struct simple_connector_data *connector_specific_data = instance->connector_specific_data;
             if (connector_specific_data->flags == NETDATA_SSL_HANDSHAKE_COMPLETE) {
-                buffer_strcat(b, "exporting|");
+                buffer_strcat(b, "exporting");
```

Comment:
is that change related to removing `NETDATA_IN_CI`?

Reply:
No, just a quick fix :-) (The `|` will be added below).

Reply:
ok, asked because of the commit name 😄